### PR TITLE
Clear list of OOB packages built on 6.0

### PR DESCRIPTION
**Pull request by safern**
Clear the list of OOB packages built for libraries on 6.0 for the February servicing release.

### CODE REVIEW

**Review by carlossanlop — APPROVED** [review]
@safern explained to me the process and reason behind this change. LGTM.

**Review by stephentoub — APPROVED** [review]

File: src/libraries/System.Text.Json/src/System.Text.Json.csproj [hunk 1]
```diff
@@ -9,7 +9,6 @@
     <Nullable>enable</Nullable>
     <IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>
     <IsPackable>true</IsPackable>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (ericstj) [inline src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: Did you want to prepare this for the PRs that @eiriktsarpalis has, or will that be a separate PR?

**Review by ericstj — APPROVED** [review]

File: src/libraries/System.Text.Json/src/System.Text.Json.csproj (hunk 1 above)
Comment (safern) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: I'd rather do that on every individual PR so that the commit contains the changes for historic reasons on what's needed to service.

Comment (eiriktsarpalis) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: We have about 7-8 fixes in the pipeline, should we be doing this separately for every PR?

Comment (safern) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: Are all for JSON?

Comment (safern) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: A ok. Given that it is multiple fixes. I'll update this PR and adjust accordingly to build a new version of the JSON Package.

Comment (eiriktsarpalis) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: Thanks 👍

Comment (safern) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: Done.

Comment (ericstj) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: I can't recall if we maintain a version for the ref-pack too or if we just let it sync with the ProductVersion.  Regardless, let's make sure that stays active in this release since I expect some changes will be in the source generator.

**Review by ericstj — APPROVED** [review]

File: src/libraries/System.Text.Json/src/System.Text.Json.csproj (hunk 1 above)
Comment (safern) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:12]: Will do that in a separate PR.
